### DISCUSSION TITLE
Welcome to FUComplete Patch Discussions!

### DISCUSSION THREAD

**Discussion by IncognitoMan** [FUComplete/Patch#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  We also ask that you follow this simple rule:
  * Do **NOT** discuss, share or link to any content related to piracy. This means no pre-patched images, unchanged game files or ISOs and anything that could be considered copyright infringement.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by slentara1** [comment]
Hey guys. I am an avid Monster Hunter. I also go by Braun and my wife, Sparrow, and I stream Monster Hunter on YouTube and Twitch. Currently we're experimenting with bringing Stories monster textures into the mainline games.

**Comment by Malkraz** [comment]
Really appreciate the love you're giving this game. Can't wait for 1.3.0!

**Comment by filminghub** [comment]
Hey guys, I've heard somewhere that you guys are planning to retool some weapons to play like later gen. Is this true?

**Comment by imeasyplease** [comment]
Hello. With the latest release I have seen that you have released a tool that can easily configure your patch. I patched my game about 2 weeks ago so I missed out on this and would simply like to adjust my game to have the DOSaudio option enabled but I am struggling with doing this. 
I would appreciate help with this issue, thanks.

**Reply by imeasyplease** [reply]
As of this moment I updated my patch to 3.0 but i still can not get dosaudio to work
there is information provided related to a file named nativepsp but i can't find such a file anywhere. the ISO folder only has the ISO and the orginal pre-patched copy

**Reply by IncognitoMan** [reply]
Did you follow this? https://github.com/FUComplete/Patch#where-do-i-place-configbin

**Comment by nachotacos69** [comment]
hello people, is there anyway to replace BGMs?, i wanna use some other MH BGMs if its possible

**Comment by palop4l** [comment]
First, i want to say thank you very much for this. IncognitoMan and team, you guys are awesome!

Second, i want to ask if it's possible to disable or remove certain parts of the UI? Like this textbox asset in in-game quick item selection. IMO it's using too much space for my own liking and sometimes hinders me from seeing stuff in bottom-right area. I initially thought to replace the textbox texture assets with blank transparent PNGs but realized the assets also used in most textbox in the game. Still i tried it and while it looks cleaner in-game, majority of texts at that point are simply painful to read.

If it's possible, it would be great if you guys add an option in CONFIG.BIN file that can enable/disable that particular aspect of UI. Once again, thank you for your awesome work!
![IMG_20241211_204823](https://github.com/user-attachments/assets/ec9084cb-eb36-4d1a-b7f4-34dbc4eb1a54)

**Reply by palop4l** [reply]
I would also like to contribute to the patch with aspect ratio cwcheats i made. Whom do i contact regarding this?

Another thing, is it possible to ask for the cwcheat version of the fov (or at leats memory address for it)? I would like to mess around with it like testing higher fov than available in the patch. Thank you so much!

**Reply by IncognitoMan** [reply]
> I would also like to contribute to the patch with aspect ratio cwcheats i made. Whom do i contact regarding this?
> 
> Another thing, is it possible to ask for the cwcheat version of the fov (or at leats memory address for it)? I would like to mess around with it like testing higher fov than available in the patch. Thank you so much!

I've done aspect ratio patches a while ago... though I guess I never got around to posting them up on the main site...


```
// Aspect ratio patches for FUComplete
// Be aware that the game has some aggressive culling or straight up
// doesn't have meshes that are rendered outside the intended aspect ratio...
// You have been warned.
// Make sure you enable "Stretch" in "Display layout & effects"
_C0 Default 16:9 - 1.764705
_L 0x1001397C 0x00003FE1
_L 0x10013980 0x0000E1E2
_C0 4:3 - 1.333333
_L 0x1001397C 0x00003FAA
_L 0x10013980 0x0000AAAB
_C0 3:2 - 1.5
_L 0x1001397C 0x00003FC0
_L 0x10013980 0x00000000
_C0 18:9 - 2
_L 0x1001397C 0x00004000
_L 0x10013980 0x00000000
_C0 20.9 - 2.222222
_L 0x1001397C 0x0000400E
_L 0x10013980 0x000038E3
_C0 21:9 - 2.333333
_L 0x1001397C 0x00004015
_L 0x10013980 0x00005555
_C0 32:9 - 3.555556
_L 0x1001397C 0x00004063
_L 0x10013980 0x00008E39
```

As for FoV, you can check the code how it's done as it'll show the address written to.

https://github.com/FUComplete/Patch/blob/f639f1e20d9ca39234b81644c5ed5c374312d6e2/modloader/preloader_exec.asm#L382-L385

Be aware though that just like the aspect ratio, there is aggressive culling... also be aware that the higher the FoV the more "fisheye" it becomes too...

**Reply by palop4l** [reply]
Great! Will check it out later..

Yeah sure enough it isn't perfect since the game isn't made for wider aspect ratio. The wider the aspect ratio like 32:9, the more unfavorable the game will look since you'll really see parts of the game that had no texture for it. But at 21:9 (at least in my code) imo the experience is mostly great and i wouldn't call the culling aggressive either but yes it's there. Probably the most jarring thing is fog in Desert area isn't scaled to cover wider aspect ratio, leaving side edge of the screen 'unfogged'. Will also test your aspect ratio codes later to see whether yours have the same issue or not👍🏻
